src/battle_animation.cpp

```diff
@@ -293,8 +293,12 @@ void BattleAnimationBattle::Draw(Bitmap& dst) {
 	for (auto* battler: battlers) {
 		const Sprite_Battler* sprite = battler->GetBattleSprite();
 		int offset = 0;
-		if (sprite && sprite->GetBitmap()) {
-			offset = CalculateOffset(animation.position, sprite->GetHeight());
+		if (sprite) {
+			if (sprite->GetBitmap()) {
+				offset = CalculateOffset(animation.position, sprite->GetHeight());
+			} else {
+				offset = CalculateOffset(animation.position, GetAnimationCellHeight() / 2);
```

Comment:
just a marker for me to test this in emscripten

Reply:
well yeah this works. As this depends on the battle sprite the sprite is usually already loaded before anyway, so no emscripten worries here